src/utils/SiteHelper.ts

```diff
@@ -21,3 +21,10 @@ export const enableModule = (moduleName: string, siteKey: string): void => {
         type: 'application/yaml'
     });
 };
+
+export const editServerName = (siteKey: string, serverName: string): void => {
```

Comment:
It would be better to have a generic function to edit a site instead of just the serverName. It could be usefull for the future.
You can pass an object as the second parameter like it's done for the create site, so it won't break the test if we modify this function